bitrepository-service/src/main/java/org/bitrepository/service/workflow/WorkflowManager.java

```diff
@@ -124,17 +126,19 @@ private void loadWorkFlows(WorkflowSettings configuration) {
                         } else {
                             collectionsToScheduleWorkflowFor = SettingsUtils.getAllCollectionsIDs();
                         }
+                        Duration workflowInterval = schedule.getWorkflowInterval();
+                        long workflowIntervalMillis = XmlUtils.xmlDurationToMilliseconds(workflowInterval);
```

Comment:
Currently, there is a small issue here that shows in the quickstart. 

Both `RepairMissingFiles-` and `SaltedChecksumWorkflow` have their interval set to `-PT1S` in the quickstart, but `XmlUtils.xmlDurationToMilliseconds(...)` doesn't handle the negation so `workflowIntervalMillis` ends up being assigned a positive value, making the `JobScheduler` start the workflow-job on a 1 second interval instead of ignoring it.

So I suppose `XmlUtils.xmlDurationToMilliseconds(...)` needs to handle negative values as well - otherwise there'd have to be a special handling here.

Reply:
I have fixed the bug in XmlUtils as you suggested.